dataeng/jobs/analytics/VerticaToBigquerySchemaCopyBusinessIntelligence.groovy

```diff
@@ -0,0 +1,33 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.from_date_interval_parameter
+import static org.edx.jenkins.dsl.AnalyticsConstants.to_date_interval_parameter
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+class VerticaToBigquerySchemaCopyBusinessIntelligence {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("vertica-to-bigquery-schema-copy-business-intelligence") {
+            logRotator common_log_rotator(allVars)
+            parameters common_parameters(allVars)
+            parameters {
+                stringParam('SCHEMA', allVars.get('SCHEMA'),'Vertica Schema')
+                stringParam('OVERWRITE', allVars.get('OVERWRITE'), 'Set to: --overwrite if you want to enable overwrite')
+                stringParam('VERTICA_CREDENTIALS', allVars.get('VERTICA_CREDENTIALS'), 'The path to the Vertica credentials file.')
+                stringParam('GCP_CREDENTIALS', allVars.get('GCP_CREDENTIALS'), 'The path to the GCP credentials file.')
+                stringParam('EXCLUDE', allVars.get('EXCLUDE'))
+                stringParam('RUN_DATE', allVars.get('RUN_DATE'))
+            }
+            multiscm common_multiscm(allVars)
+            triggers common_triggers(allVars)
+            wrappers common_wrappers(allVars)
+            publishers common_publishers(allVars)
+            steps {
+                shell(dslFactory.readFileFromWorkspace('dataeng/resources/vertica-to-bigquery-schema-copy-business-intelligence.sh'))
```

Comment:
I would suggest renaming the script to remove business-intelligence from the name.   If we want a second job to copy another schema (e.g. finance or curated), then we should be able to use the same script with a different SCHEMA argument.